src/main/haskell/kore/app/exec/Main.hs

```diff
@@ -201,6 +202,7 @@ data KoreExecOptions = KoreExecOptions
     , koreLogOptions      :: !KoreLogOptions
     , koreSearchOptions   :: !(Maybe KoreSearchOptions)
     , koreProveOptions    :: !(Maybe KoreProveOptions)
+    , koreRepl            :: !Bool
```

Comment:
I was already uncomfortable with having a single entry point (`kore-exec`) for execution, search, and verification. How would you feel about separating the REPL into a different executable? I think it would simplify handling the command-line, and it would further encourage breaking up functionalities and including them in the library.

Reply:
I agree with that, however the `repl` currently only does anything for verification, so in essence I would have to extract the verification code, which seemed like too big of a change for this task. I think I'd rather wait and see where this goes before committing to any splitting (since it would also require changes in other places that already use `kore-exec` for both execution and verification).